.github/workflows/php.yml

```diff
@@ -0,0 +1,226 @@
+      - name: "Build Notification"
+        id: slack
+        uses: slackapi/slack-github-action@v1.19.0
+        with:
+          channel-id: 'G3184HDT6' # CICD
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "${{ needs.build.result == 'success' && '#2eb886' || '#a30200' }}",
+                  "blocks": [
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "*${{ needs.build.result == 'success' && 'Build Success' || 'Build Failure' }}: ${{github.event.repository.name}}*"
+                      }
+                    },
+                    { "type": "divider" },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "Build <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|#${{ github.run_number }}> by ${{github.actor}} ${{ needs.build.result == 'success' && 'successful' || 'failed' }}! ${{ needs.build.result == 'success' && ':ok_hand:' || ':scream::cry:' }}\n\nCommit Message:\n${{ steps.slack-prepare.outputs.commit-message }}"
+                      },
+                      "accessory": {
+                        "type": "image",
+                        "image_url": "${{ needs.build.result == 'success' && 'https://styles.redditmedia.com/t5_39983v/styles/communityIcon_nj9g8x1mm3t51.png?width=256&s=957cfecf8859a834aa7d7094a64b791f6f4ec0a0' || 'https://i.pinimg.com/originals/11/a5/57/11a557b9e90647df60af69d865084ec9.jpg' }}",
+                        "alt_text": "${{ needs.build.result == 'success' && 'Nice!' || 'Oof.' }}"
+                      }
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": ":github: *Repository*: <${{ github.server_url }}/${{ github.repository}}|${{ github.repository }}>\n:git: *Ref*: ${{ github.ref_name}}\n:1234: *Build Number*: <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|#${{ github.run_number }}>\n:docker: *Image Tag*: ${{needs.build.outputs.image_tag_build}}"
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+
+  deploy:
+    name: "Deploy"
+    if: contains(fromJson('["development", "staging", "production"]'), github.ref_name)
+    runs-on: ubuntu-latest
+    needs: [build]
+    strategy:
+      fail-fast: true
+      matrix:
+        container: [ "php", "nginx" ]
+    steps:
+
+      - name: "Install Kubectl"
+        uses: azure/setup-kubectl@v2.0
```

Comment:
Wonder if we need to specify a `kubectl` version here as it tends to complain if `n+/-1` where client behavior is not guaranteed...
```
WARNING: version difference between client (1.24) and server (1.22) exceeds the supported minor version skew of +/-1
```

May be good to make a new ticket to tackle that specifically for compat.

Reply:
It's as easy as adding a version param :-) 
https://github.com/Azure/setup-kubectl

Ticket added:
https://linced.atlassian.net/browse/DEVOPS-145